components/policy/resources/templates/policy_definitions/README.md

```diff
@@ -0,0 +1,22 @@
+## Policy Settings in Brave
+
+These are Brave specific group policies.
+
+Our official documentation is [available here](https://support.brave.com/hc/en-us/articles/360039248271-Group-Policy).
+
+For information about adding a new policy, please see [this page](https://source.chromium.org/chromium/chromium/src/+/main:docs/enterprise/add_new_policy.md).
+
+In a nutshell, the steps for adding a new policy in Brave look like this:
+
+1. Create a new .yaml file under `src/brave/components/policy/resources/templates/policy_definitions/BraveSoftware/`. The name of the file itself will be the policy name. Chromium uses capital casing. For example, we have a policy for disabling Brave Rewards called `BraveRewardsDisable.yaml`. The name used for matching is `BraveRewardsDisable`.
+2. Update the properties in that file accordingly. Look at some of the existing ones as an example.
+3. Go into `src/brave/browser/policy/brave_simple_policy_map.h` and add your entry here. It'll be auto-generated as `policy::key::k` and then the policy name. With the above example, that would be `policy::key::kBraveRewardsDisable`. You must map this to a profile preference. That is what you'll check in the code.
+4. Go into `src/brave/chromium_src/components/policy/resources/policy_templates.py` and add your policy name to the end of the existing string array there.
+5. In the code where you want to check the profile preference, you can tell if it's set via policy by checking `prefs->IsManagedPreference(<profile_preference_key_here>)`. If this is set to true, you might want to have the UI display something like "This setting is managed by your organization" and have it be read-only.
+6. Build the project like regular (`npm run build`)
```

Comment:
see, the build instruction is already wrong, because you don't mention that you need to run `npm run sync` to copy new policies to the upstream dir.

pls try the `main()` override in `policy_templates.py` and see how it works.

Reply:
This approach seems to be working great! I'll wrap up testing tomorrow (for me; Thursday) and update this documentation